launch/test/launch/utilities/test_type_utils.py

```diff
@@ -283,8 +285,10 @@ def test_coercion_raises_value_error(coerce_to_type_impl):
         coerce_to_type_impl('', data_type=bool)
     with pytest.raises(ValueError):
         coerce_to_type_impl('Bsd', data_type=bool)
-    with pytest.raises(ValueError):
-        coerce_to_type_impl('1', data_type=bool)
+
+    # '1' and '0' are special cases for bool coercion
+    # with pytest.raises(ValueError):
+    #     coerce_to_type_impl('1', data_type=bool)
```

Comment:
I think it's okay to remove these commented lines.

Reply:
Okie https://github.com/ros2/launch/pull/651/commits/89847240d2072199d8ae5dafcb82123cb2418323